packages/api/tests/endpoints/test-granules.js

```diff
@@ -1742,6 +1742,26 @@ test.serial('create (POST) creates new granule with associated execution in dyna
   t.is(fetchedESRecord.granuleId, newGranule.granuleId);
 });
 
+test.serial('create (POST) publishes an SNS message upon successful granule creation', async (t) => {
+  const newGranule = fakeGranuleFactoryV2({
```

Comment:
I would have used fakeGranuleRecordFactory but this fails with a bad request error since we are reading from DynamoDB and not PG here https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/endpoints/granules.js#L81-L87 

A ticket was already created to address updating this https://bugs.earthdata.nasa.gov/browse/CUMULUS-2759

Reply:
Not sure if I should wait on that ticket or just leave this unit test as is

Reply:
I think that you can leave it as is